src/docs/user/ProActiveUserGuide.adoc

```diff
@@ -1562,16 +1562,20 @@ The next parameters are provided as task variables:
 |===
 |Parameter Name | Description | Example
 |ldapURL|URL the LDAP server|`ldap://localhost:389` (unsecured) or `ldaps://127.0.0.1:636` (secured)
-|ldapSearchBase|base domain name for LDAP queries|`dc=yourOrganization,dc=com`
+|ldapDnBase|The base distinguished name of the catalog| `dc=yourOrganization,dc=com`
+|ldapUsername|The username | `cn=admin` for global admin or `cn=yourName,ou=users` for a specific user
+|ldapSearchBase|base domain name for ldap queries|`dc=specificDepartment` or can be empty for base DN search
 |ldapSearchFilter|LDAP search filter|`(objectclass=*)`
 |ldapSelectedAttributes|list of attributes to retrieve|`attributeName1,attributeName2`
+
 |===
 
+If any of these variables must stay private you'll need to set them as third-party credentials (details are given below) instead.
+
 ==== Authenticate with Third-party credentials
 
-In order to be able to authenticate to the LDAP server you need to define the right Scheduler user in the <<_third_party_credentials,Third-party credentials>>: 
+In order to be able to authenticate to the LDAP server you need to define for the right Scheduler user in the <<_third_party_credentials,Third-party credentials>>: 
```

Comment:
It was not relevant to display a list for one item so I replace the sentence by naming the item.